### DISCUSSION TITLE
The copied text missing trailing spaces

### DISCUSSION THREAD

**Discussion by lykahb** [ghostty-org/ghostty#2861] [Q&A] [answered]
Select the text like this, selection is marked between `>`, `<`

```
$ echo "start, text with significant trailing space       , end"
start, >text with significant trailing space       <, end
```
The text in the clipboard is missing the trailing space. If I copy just the spaces, the clipboard is empty.

My actual use case that this behavior broke was to copy an ideographic space character from the output.

Edit: it's not clear if this is a bug or a fancy convenience feature. I wouldn't expect the tools like terminal or IDE to do this

**Comment by mitchellh** [comment, marked as answer]
I believe that our selection copy does currently trim all whitespace, but I think it only trims ASCII space and tab. It shouldn't be trimming any other unicode codepoint.  I haven't heard anyone complain about it yet to this point. I'd be curious to hear from others about this.

We could always introduce a separate keybinding action to also include whitespace.
